src/routes/adminRoutes.js

```diff
@@ -5,5 +5,7 @@ const adminController = require('../controllers/adminController');
 // Define the admin routes
 router.post('/create-user', adminController.verifyAdmin, adminController.createUser);
 router.delete('/delete-user/:uid', adminController.verifyAdmin, adminController.deleteUser);
+router.get('/users', adminController.verifyAdmin, adminController.getUsers);
+router.get('/users/:uid', adminController.verifyAdmin, adminController.getUserByUid);
```

Comment:
See original MR comments, but none of these routes should exist. These should all be routes under `/user` and if we want to require an admin token for them we can do it there.